### PR TITLE
Fix for duplicate event names

### PR DESCRIPTION
**Pull request by CodingSam**
## Description :sparkles:
Add fix for duplicate event names

## Additional notes :spiral_notepad:

### CODE REVIEW

**Comment by CodingSam** [comment]
Faker has not published a version to npm where the unique function is included. So I made a small recursive function.

**Review by nenonja — APPROVED** [review]
Looks good!

File: src/util/mockDataUtils.ts [hunk 1]
```diff
@@ -46,11 +46,25 @@ export const fakeEvents = (
...
+
+    return name;
+  };
+
   return merge(
     {
       id: `hel:${faker.random.uuid()}`,
       internalId: faker.random.uuid(),
-      name: fakeLocalizedObject(faker.name.title()),
+      name: fakeLocalizedObject(uniqueName()),
```
Comment (Sartonon) [inline src/util/mockDataUtils.ts:67]: Is there same risk of having duplicates in other tests too?

Comment (CodingSam) [reply src/util/mockDataUtils.ts:67]: Hmm, maybe? What if we went like this now, and if in the future if we still get this error, then we could make a utility which could compare all the fields and make sure there are no duplicates.

**Pull request merged**